sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -396,6 +432,7 @@ internal BlobBaseClient(Uri blobUri, HttpPipelinePolicy authentication, BlobClie
         /// <returns>A new <see cref="BlobBaseClient"/> instance.</returns>
         protected virtual BlobBaseClient WithSnapshotCore(string snapshot)
```

Comment:
Do you have a withVersion method?

Reply:
There isn't one.

Reply:
There is a `WithVersionCore` below if that needs similar treatment.

Reply:
Oh I'm blind there is one